src/control/Control.Layers.js

```diff
@@ -228,6 +228,7 @@ export const Layers = Control.extend({
 		}, this);
 
 		if (!collapsed) {
+			section.setAttribute('aria-label', link.title);

```

Comment:
Why only do this when the layer is not collapsed? The visibility of an element should not necessarily influence the textual content, unless it's something explicitly related to announcing the visibility status.